client/src/legacy/LeftAndMain.FieldHelp.js

```diff
@@ -10,28 +10,40 @@ $.entwine('ss', function($) {
    * input fields (e.g. GridField), and aren't compatible
    * with showing tooltips.
    */
-  $(".cms .field.cms-description-tooltip").entwine({
+  $(".cms .field.cms-description-tooltip .form__field-holder").entwine({
     onmatch: function() {
       this._super();
 
-      var descriptionEl = this.find('.description'), inputEl, tooltipEl;
+      var descriptionEl = this.find('.form__field-description');
```

Comment:
I'm not 💯 on top of our conventions, but shouldn't classes that trigger JS behaviour not be BEM classes (ie: not style classes)?

I'd be using `.js-tooltip`

Reply:
it's a debatable point, I think we still allow for (and use) BEM classes for JS behaviour all over the codebase.

No convention/practice has really stuck - e.g. trying to prefix `js-` for function/js classes got cumbersome.
I think in the end, as long as the class communicates the purpose it has (the whole point of BEM), it's fine 👍